src/js/worker/core/player.js

```diff
@@ -884,18 +922,42 @@ function generate(
     scoutingRank: number,
 ): PlayerWithoutPid {
     let ratings;
+	
+	var PD = require("probability-distributions");	
+	let realHeight;
+	let predetHgt;
+    
+	// RealHeight will never go lower than 4'0" or higher than 9'6".  (Very unlikely to ever see those anyway.)	
+	realHeight = helpers.bound(PD.rgamma(1,472.0530111,5.981338), 48, 114);
+	
+	// Ugly hack: extreme outliers (7'6"+) are about right, but it's generating too many guys 7-3 to 7-5
+	// So lose two inches on some of them
```

Comment:
I don't know why I didn't think of this sooner, but there is an elegant solution here.  With so many data points, it's enough to just build a custom distribution.  Solves two problems for you.  You won't need to include the distributions package to get the gamma function, and it will remove this ugly hack.